tr.yml

```diff
@@ -0,0 +1,279 @@
+  EXITDLG_BUTTON_EXIT: "&Windhawk'tan çık"
+  EXITDLG_BUTTON_CANCEL: "İptal"
+  NOTIFICATION_UPDATE_APP: "Windhawk'ın yeni bir sürümü mevcut"
+  NOTIFICATION_UPDATE_APP_MOD: "Windhawk'ın yeni bir sürümü mevcut ve bir mod güncellenebilir"
+  NOTIFICATION_UPDATE_APP_MODS: "Windhawk'ın yeni bir sürümü mevcut ve %d mod güncellenebilir"
+  NOTIFICATION_UPDATE_MOD: "Bir Windhawk modu güncellenebilir"
+  NOTIFICATION_UPDATE_MODS: "%d Windhawk modu güncellenebilir"
+  TRAYICON_TOOLTIP_UPDATE: "Yeni bir sürüm mevcut"
+  TASKDLG_TITLE_LOADED_MODS: "Yüklü modlar"
+  TASKDLG_TITLE_TASKS_IN_PROGRESS: "Mod görevleri devam ediyor"
+  TASKDLG_BUTTON_OPEN_APP: "&Windhawk'ı aç"
+  TASKDLG_COLUMN_MOD: "Mod"
+  TASKDLG_COLUMN_PROCESS: "İşlem"
+  TASKDLG_COLUMN_PID: "PID"
+  TASKDLG_COLUMN_STATUS: "Durum"
+  TASKDLG_STATUS_PENDING: "Beklemede..."
+  TASKDLG_STATUS_LOADING: "Yükleniyor..."
+  TASKDLG_STATUS_LOADED: "Yüklendi"
+  TASKDLG_STATUS_UNLOADED: "Yüklenmedi"
+  TASKDLG_TASK_INITIALIZING: "Başlatılıyor..."
+  TASKDLG_TASK_LOADING_SYMBOLS: "Semboller yükleniyor..."
+app:
+  general:
+    loading: "Yükleniyor..."
+    loadingFailed: "Yükleme başarısız, lütfen internet bağlantınızı kontrol edin"
+    loadingFailedTitle: "Yükleme başarısız"
+    loadingFailedSubtitle: "Lütfen internet bağlantınızı kontrol edin ve tekrar deneyin"
+    tryAgain: "Tekrar dene"
+    updating: "Güncelleniyor..."
+    installing: "Yükleniyor..."
+    compiling: "Derleniyor..."
+  appHeader:
+    home: "Ana Sayfa"
+    explore: "Keşfet"
+    settings: "Ayarlar"
+    about: "Hakkında"
+  mod:
+    updateAvailable: "Güncelleme mevcut"
+    installed: "Yüklendi"
+    details: "Ayrıntılar"
+    remove: "Kaldır"
+    removeConfirm: "Bu modu kaldırmak istediğinizden emin misiniz??"
+    removeConfirmOk: "Modu kaldır"
+    removeConfirmCancel: "İptal"
+    notCompiled: "Modun derlenmesi gerekiyor"
+    editedLocally: "Mod yerel olarak düzenlendi"
+    noDescription: "(açıklama yok)"
+    users_one: "{{formattedCount}} kullanıcı"
+    users_other: "{{formattedCount}} kullanıcı"
+  modDetails:
+    header:
+      installedVersion: "Yüklü sürüm"
+      latestVersion: "En son sürüm"
+      loading: "Yükleniyor..."
+      loadingFailed: "yükleme başarısız"
+      modId: "mod tanımlayıcı"
+      modVersion: "Mod sürümü"
+      modAuthor:
+        title: "Mod yapımcısı"
+        homepage: "Ana Sayfa"
+        github: "GitHub"
+        twitter: "Twitter"
+      processes:
+        all: "Tüm işlemler"
+        allBut: "All but {{list}}"
+        except: "{{included}} except {{excluded}}"
+        tooltip:
+          target: "Hedef işlem"
+          targets: "Hedef işlemler:"
+          excluded: "Hariç::"
+      update: "Güncelle"
+      updateNotNeeded: "Yüklü sürüm, en son sürümle aynı"
+      install: "Yükle"
+      compile: "Derle"
+      disable: "Devre dışı bırak"
+      enable: "Etkinleştir"
+      edit: "Düzenle"
+      fork: "Fork"
```

Comment:
Untranslated on purpose? Perhaps it should be [çatal/çatallanarak](https://tr.wikipedia.org/wiki/%C3%87atal_(yaz%C4%B1l%C4%B1m_geli%C5%9Ftirme)).